packages/plugins/i18n/admin/src/hooks/useI18n.ts

```diff
@@ -43,14 +43,15 @@ const useI18n: UseI18n = () => {
     );
   }, [params.slug, userPermissions]);
 
+  // TODO: use specific hook to get schema only
   const { schema } = useDocument(
     {
       // We can non-null assert these because below we skip the query if they are not present
       collectionType: params.collectionType!,
       model: params.slug!,
     },
     {
-      skip: !params.slug || !params.collectionType,
+      skip: true,
```

Comment:
Why do we now always skip? Also you can use selectFromResult if you want more specific

Reply:
that skip option doesn't allow skipping the schema loading but only skipping a document data here there is never a reason to load a document data. 

Overall that hook should not even be used everywhere like it is right now it creates so much confusion on what it does it's very unsafe